modrob0:
  # Publish all joint states --------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100

  # Position Controllers ------------------------------------
  joint0_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint0
    pid: {p: 5000.0, i: 0.0, d: 112.0}
  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint1
    pid: {p: 100000.0, i: 0.0, d: 0.0}
  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint2
    pid: {p: 700.0, i: 0.0, d: 0.0}
  hand_to_finger1_position_controller:
    type: effort_controllers/JointPositionController
    joint: hand_to_finger1
    pid: {p: 1000.0, i: 0.0, d: 0.0}
  hand_to_finger2_position_controller:
    type: effort_controllers/JointPositionController
    joint: hand_to_finger2
    pid: {p: 1000.0, i: 0.0, d: 0.0}
